### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nqst**
- Fixed syntax highlighting
- Updated an alert classname to the new one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ production:
 
 Refer to the <dfn>Templates</dfn> (which you have set in the [config](https://github.com/transloadit/rails-sdk/#configuration)) with the `transloadit` helper.
 
-```html
+```ruby
```
Comment (rmehner) [inline README.md:93]: This is actually `erb`

Comment (nqst) [reply README.md]: Valid point! Just fixed, thanks 👍

**Comment by rmehner** [comment]
Thank you @nqst and hope you stay safe!

**Comment by nqst** [comment]
Thank you so much! 🇺🇦

**Pull request merged**